web/templates/pages/tasks.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Tasks{% endblock %}
+
+{% block content%}
+
+	<header class="row">
+		<div class="col-lg-6 col-sm-7">
+		      <h1>All CodeCatz Tasks</h1>
+		</div>
+	</header>
+
+	{% if task_list %}
+
+		<div class="row">
+
+			{% for task in task_list %}
+
+				<div class="col-md-3 col-xs-6">
+					<h3><strong>Task title: </strong>{{ task.title }}</h3>
+					<p><strong>Assigned to: </strong> {{ task.assigned_id }}</p>
+					<p><strong>Part of project: </strong><a href= "{% url 'web.views.project_detail' project_id=task.project_id.project_id %}"> {{ task.project_id.project_name }} </a></p>
+					<br></br>
```

Comment:
this can be replaced with \<br />

in principle \<br> should be enough, but the above form makes it XHTML compatible